src/model/RacingGame.js

```diff
@@ -0,0 +1,62 @@
+const RacingCar = require('./RacingCar');
+
+class RacingGame {
+  #cars;
+  #winners;
+
+  constructor(names) {
+    this.#cars = [];
+    for (const name of names) {
+      this.#cars.push(new RacingCar(name));
+    }
+    this.#winners = [];
+  }
+
+  move() {
+    for (const car of this.#cars) {
+      car.move();
+    }
+  }
+
+  getCarNames() {
+    const carNames = this.#cars.map((car) => {
+      return car.getName();
+    });
+    return carNames;
+  }
+
+  getCarMoveResults() {
+    const carMoveResults = this.#cars.map((car) => {
+      return car.getMoveResult();
+    });
+    return carMoveResults;
+  }
+
+  calculateWinners() {
+    const carNames = this.getCarNames();
+    const carMoveResults = this.getCarMoveResults();
+    const carMoveResultLengths = getArrayLengths(carMoveResults);
+    const winningScore = Math.max(...carMoveResultLengths);
+    this.#setWinners(carNames, carMoveResultLengths, winningScore);
+  }
+
+  #setWinners(names, moveLengths, winningScore) {
+    names.forEach((element, idx) => {
+      if (moveLengths[idx] === winningScore) {
+        this.#winners.push(element);
+      }
+    });
+  }
+
+  getWinners() {
+    return this.#winners;
+  }
+}
+
+const getArrayLengths = (array) => {
+  return array.map((element) => {
+    return element.length;
+  });
+};
```

Comment:
```suggestion
const getArrayLengths = array => array.map(element => element.length);
```
필요없어진 함수긴 하지만 그래도 깔끔하게 두는게 낫지. 화살표 함수에서 return 생략할 수 있으면 생략하는게 더 깔끔해